docs/reference/module-api/module-options.md

```diff
@@ -125,6 +126,34 @@ module.exports = {
 }
 ```
 
+### `i18n`
+
+If set to a truthy value, Apostrophe will look for an `i18n` directory *in that module* with one or more JSON files with localization string definitions (e.g., `modules/project-languages/es.json`). The JSON file names should match configured locale names. [See the static i18n guide](TODO) for more information.
+
+If set to an object, there may be `ns` and `browser` properties.
+
+| `i18n` setting | Value type | Description |
+|---------|---------|---------|
+| `ns` | String | A namespace for localization string keys in this module. If undefined, Apostrophe will use the `'default'` namespace. That namespace is intended for project-level localization. |
```

Comment:
The namespace `'apostrophe'` is reserved for Apostrophe's UI. You may intentionally set `ns` to `'apostrophe'` if your goal is to localize the Apostrophe user interface.